client/src/modules/dashboard/Dashboard.scss

```diff
@@ -1,43 +1,5 @@
 .dashboard {
-  margin-top: 36px;
-
-  &__heading {
-    display: flex;
-    align-items: center;
-    font-size: 18px;
-    line-height: 1.2;
-    margin-bottom: 16px;
-    color: $text-normal;
-
-    svg {
-      height: 18px;
-      margin-right: 16px;
-
-      path {
-        fill: $text-normal;
-      }
-    }
-  }
-
-  &__list {
-    margin: 0 -10px 32px;
-    display: grid;
-    grid-template-columns: repeat(auto-fill, 200px);
-    grid-gap: 20px 10px;
-  }
-
-  &__list-item {
-    margin-left: auto;
-    margin-right: auto;
-
-    a {
-      text-decoration: none;
-    }
-  }
-
-  &__list-link {
-    text-decoration: none;
-  }
+  @include cohorts-lists-view;
```

Comment:
this looks like some more general style - name it differently and place e.g. in commons

Reply:
Because this styles and js code are used  few times in different places I created a new component for them